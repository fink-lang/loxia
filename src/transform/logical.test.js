import {fink2js} from '../testing';


test('compiles logical', ()=> {
  expect(
    fink2js(`
      logical = a || b || c && d < 1 <= s
    `)
  ).toMatchSnapshot();
});


