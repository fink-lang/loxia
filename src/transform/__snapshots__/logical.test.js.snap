// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compiles logical 1`] = `
"const logical = a || b || c && d < 1 <= s;
Object.assign(module.exports, {
  logical
});"
`;
